set_up_interface.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
@blyscuit What do you think about using `swift` instead of bash?
```suggestion
#!/usr/bin/swift
```

Reply:
Let me look in to that.

Reply:
This's optional

Reply:
Will create a task for that. Although I don't think Swift will support replace all in files like `sed -I` so we will need to create some extension for it, which isn't ideal as it adds more failure points.

Reply:
@blyscuit Yes, that's a good point!. We could create POC first before having a task